src/TimeColumn.js

```diff
@@ -129,7 +130,14 @@ export default class TimeColumn extends Component {
     // excluding the time gutter TimeColumn from having a time indicator.
     if (!dragThroughEvents) return;
 
-    const now = dates.now(nowTimezone);
+    let now = dates.now(nowTimezone);
+
+    const daylightSavingsShift = getDaylightSavingsShift(now)
+    const isFallingBack = isDaylightSavingsFall(daylightSavingsShift);
+
+    if (isFallingBack) {
+      now = dates.add(now, daylightSavingsShift, 'minutes');
+    }
```

Comment:
I'm not sure this is the right condition - don't we need to check whether the current time is pre-time change or post-time change (i.e. 2AM)? This code will always shift the indicator position as long as it's on a Sunday of fall daylight savings, but my understanding is that the indicator is only positioned incorrectly on one side of that time change.

Reply:
Thanks! Yeah, it should only be off after 2:00AM. I'll update the condition.